src/stylesheets/components/_layout.scss

```diff
@@ -1,36 +1,15 @@
 // Flexbox positioning to move sidenav below main content on small screens
-.usa-layout-docs {
-  display: flex;
-  flex-direction: column;
-
-  @include media($large-screen) {
-    display: inherit;
-  }
-
-  .usa-grid {
-    > :first-child:not(.usa-width-*) {
-      margin-top: 0;
-    }
-
-    > :last-child:not(.usa-width-*) {
-      margin-bottom: 0;
-    }
-  }
-}
-
 .usa-layout-docs-sidenav {
   order: 2;
-}
-
-.usa-layout-docs-main_content {
-  margin-bottom: $site-margins;
-  order: 1;
+  padding-top: units(6);
```

Comment:
@thisisdano how much space should we have between the bottom of the page and the nav below?

Reply:
4 units seems like a good place to start

Reply:
Updated in 6e52d90, thanks! Also, this PR is ready to review 👓